legacy/full/Dockerfile

```diff
@@ -184,7 +184,7 @@ LABEL dazzle/layer=lang-rust
 LABEL dazzle/test=tests/lang-rust.yaml
 USER gitpod
 RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
-    curl -fsSL https://sh.rustup.rs | sh -s -- -y --profile minimal --default-toolchain 1.58.0 \
+    curl -fsSL https://sh.rustup.rs | sh -s -- -y --profile minimal --default-toolchain 1.56.1 \
```

Comment:
This was on behalf of the commit revert, just FYI.

Reply:
There should be absolutely no change in the `/legacy` dir. Please rebase/squash your commits so there is no changes in this PR wrt to legacy dir

Reply:
Got it, my bad, there might have been some conflict. Will revert that change.